### PR TITLE
feat: rar e2e

### PR DESCRIPTION
**Pull request by ReedSoftware**

### CODE REVIEW

File: server/routes/journeys/record-review/e2e.cy.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    cy.contains('dt', 'Attended in person').next().should('include.text', 'Yes')
+    cy.contains('dt', 'Contribution').next().should('include.text', 'a contrib')
+
+    verifyCyaChangeLinks()
+  }
+
+  const verifyCyaChangeLinks = () => {
+    cy.findByRole('link', { name: /Change the details of the review/ }).click()
+    cy.url().should('to.match', /details#summary$/)
+    cy.get('textarea').clear().type('changed summary', { delay: 0 })
```
Comment (JackReeveMoJ) [inline server/routes/journeys/record-review/e2e.cy.ts:86]: Is it worth using `cy.focused()` for these just as a sanity check that the input element is being focused?

Comment (ReedSoftware) [reply server/routes/journeys/record-review/e2e.cy.ts:86]: @JackReeveMoJ Why do we care about the text area being focused when we type? Im confused 😅

Comment (bill-moj) [reply server/routes/journeys/record-review/e2e.cy.ts:86]: cya page a/c says something about focusing on the specific html input element.
in the unit-esque cypress test for just the check-answers page, we didn't test its interactions with the other pages. 
so i guess that would belong to the full e2e test here if we're concerned about that.

Comment (ReedSoftware) [reply server/routes/journeys/record-review/e2e.cy.ts:86]: Oh you mean when we click the change link - that sort of detail really should belong in the check-answers tests, not this one. Added there :)

**Review by JackReeveMoJ — APPROVED** [review]

**Pull request merged**